cli/azd/cmd/templates.go

```diff
@@ -129,7 +129,7 @@ func (tl *templateListAction) Run(ctx context.Context) (*actions.ActionResult, e
 		columns := []output.Column{
 			{
 				Heading:       "Repository Path",
-				ValueTemplate: "{{.RepositoryPath}}",
+				ValueTemplate: output.WithHyperlink("https://github.com/"+"{{.RepositoryPath}}", "{{.RepositoryPath}}"),
```

Comment:
IIRC, `RepositoryPath` can be a full url (for example, to AzureDevOps or some other Git-server). The link would be broken for such cases.

Maybe you need some function within the string-template to check if the path is just a slug (repoOwner + name) or if it is a full url.

Reply:
We have the Absolute function, let me check if it works for both and use it.

Reply:
@vhvb1989 I don't know if you have checked my recent push or not but I already figured this out.

Reply:
> If you still want me to do it like you I will but let me know what you think of the current final state

No need to.  I was missing the `Transformer` option we have there.